test/language/line-terminators/line-separator-string-literal.js

```diff
@@ -0,0 +1,11 @@
+// Copyright 2018 Mathias Bynens. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-literals-string-literals
+description: >
+  LINE SEPARATOR (U+2028) is allowed within string literals.
+features: [json-superset]
+---*/
+
+eval("'foo\u2028bar\u2028baz\u2028qux'");
```

Comment:
this should go a bit beyond here. We don't need eval for these new tests, that's only legacy we don't need to import from the removed file.

At the same time, it's also a good opportunity to duplicate these tests with the literal cases using unescaped and escaped forms in each test file. I can work in a follow up if it's such a burn to add more here, let me know and I'd be glad to help.

Reply:
I used `eval` to avoid having to put the raw U+2028 and U+2029 characters in the source code, which is hard to review and would be easy to accidentally break in the future. This approach seemed more maintainable.

Reply:
`eval` is annoying because it makes these tests not really consumable by parsers and other consumers which don't implement `eval`. (See #1356 etc.) I think it's worth having the literal character in the source.